tools/cldr-code/src/main/resources/org/unicode/cldr/util/downgrade.txt

```diff
@@ -12,6 +12,9 @@
 
 # CLDR-16228 Add the paths where spaces changed in cjk+ person names
 # CLDR-16568 Require checking of questionable PNF data
+# CLDR-16534 Downgrade paths for renaming Islamic Calendar to Hijri Calendar
```

Comment:
I think we want to move this up, so the comment about PNF data is just before the PNF paths

Reply:
I've just merged the commit that moves this file from util to util/data

Reply:
> I think we want to move this up, so the comment about PNF data is just before the PNF paths

@macchiati I rearranged the lines as requested (I think)

Reply:
yes, different topic, however it's the same file so potential merge conflict

Reply:
@macchiati OK I have resolved conflicts, please approve, thanks